README.adoc

```diff
@@ -110,3 +108,25 @@ dependencies {
     implementation "care.data4life.hc-util-sdk-kmp:util-android:$version"
 }
 ----
+
+== Roadmap
+
+This project is work in progress. We are working on adding more functionality, guidelines, documentation and other improvements.
+
+== Contributing
+
+You want to help or share a proposal? You have a specific problem?
```

Comment:
I guess here is missing something like `then open up a pull request`.

Reply:
Right, that's missing